### PR TITLE
Make enable_iso not dependent on enable_rpm 

### PR DESCRIPTION
**Pull request by sean797**

### CODE REVIEW

**Comment by jlsherrill** [comment]
Commit message seems a little strange

**Comment by sean797** [comment]
Thanks @jlsherrill Updated, there was a couple more changes needed anyway.

File: examples/pulp_isorepo.pp [hunk 1]
```diff
@@ -3,6 +3,8 @@
   # https://github.com/theforeman/puppet-pulp/issues/138
   ssl_username => '',
   enable_admin => true,
+  enable_iso => true,
+  enable_rpm => false,
```
Comment (ekohl) [inline examples/pulp_isorepo.pp]: lint would like you to align these

**Comment by ekohl** [comment]
Thanks!

**Pull request merged**